requirements/docs.txt

```diff
@@ -6,7 +6,7 @@ pandoc
 docutils
 sphinxcontrib-fulltoc
 sphinxcontrib-mockautodoc
-https://github.com/PyTorchLightning/lightning_sphinx_theme/archive/master.zip#egg=pt-lightning-sphinx-theme
+https://github.com/ydcjeff/lightning_sphinx_theme/archive/method-css.zip#egg=pt-lightning-sphinx-theme
```

Comment:
we can't use a theme linked to your account... it has to be under PL.
@Borda

Reply:
@williamFalcon  Yes, I know.
It's for first time CI preview build.

After https://github.com/PyTorchLightning/lightning_sphinx_theme/pull/12 this is merged, I will change back to PL.

Reply:
merged 🎉